_posts/2017-01-19-new-pulp-logo.md

```diff
@@ -0,0 +1,30 @@
+---
+title: New Pulp Logo
+author: David Davis
+---
+
+![New Pulp Logo](/images/pulp_logo_big.png)
```

Comment:
So I'm using an absolute path which should render properly once we publish this to pulpproject.org. I could use a relative path (`../../../images/pulp_logo_big.png`). It looks kinda ugly though. What do you think?